db/migrate/20230920093000_move_ct_command_into_own_setting.rb

```diff
@@ -0,0 +1,24 @@
+class MoveCtCommandIntoOwnSetting < ActiveRecord::Migration[6.1]
+  def up
+    ['ct', 'fcct'].each do |setting_prefix|
+      setting = Setting.where(name: "#{setting_prefix}_command")
+      if setting
+        setting.value = setting.value[1..]
```

Comment:
this throws away the currently configured path and I did not add code to set up `(fc)ct_location` as I think an upgrade warning is sufficient.